src/MovieCard.jsx

```diff
@@ -1,27 +1,72 @@
-// import { useState } from 'react'
 import './MovieCard.css'
 import PropTypes from 'prop-types'
 
-const MovieCard = ({id, title, rating, poster_path, onMovieCardClick}) => {
+const MovieCard = ({id, title, rating, poster_path, onMovieCardClick, onHandleLikeClick, onHandleWatchedClick}) => {
 
   const onCardClick = () => {
     document.getElementById('modal-overlay').style.display = 'block';
     document.getElementById('modal-overlay').scrollIntoView({behavior: 'smooth'});
     onMovieCardClick(id);
   }
 
+  let clickedLikeBefore = false;
+  const handleLikeClick = (event) => {
+    event.stopPropagation();
+
+    let simplifiedMovieObject = {
+      "id": id,
+      "title" : title,
+      "vote_average": rating,
+      "poster_path": poster_path,
+    }
```

Comment:
This is duplicated! You could instead define this once outside of the function. Or better yet, only pass in the `id` to the function rather than an entire object. Then in the parent component you can do a look-up using the id